Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Haley L. Ellis",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Na Qu",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nabeel Bardeesy",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Miguel N. Rivera",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Vikram Deshpande",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Cristina R. Ferrone",
+                "orcid": "https://orcid.org/0000-0002-9465-7008",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Ravi Kapur",
+                "orcid": "",
+                "affiliation": " Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112